docs/workflow/trimming/feature-switches.md

```diff
@@ -13,6 +13,7 @@ configurations but their defaults might vary as any SDK can set the defaults dif
 | EnableUnsafeBinaryFormatterSerialization | System.Runtime.Serialization.EnableUnsafeBinaryFormatterSerialization | BinaryFormatter serialization support is trimmed when set to false |
 | EventSourceSupport | System.Diagnostics.Tracing.EventSource.IsSupported | Any EventSource related code or logic is trimmed when set to false |
 | InvariantGlobalization | System.Globalization.Invariant | All globalization specific code and data is trimmed when set to true |
+| HybridGlobalization | System.Globalization.Hybrid |  Loading ICU + native implementations |
```

Comment:
Are we still planning to add the feature switch `HybridGlobalization`? Reading up on https://github.com/dotnet/designs/blob/main/accepted/2020/feature-switch.md a bit more, it seems like feature switches primarily help control the size of libraries by trimming out areas we know will not be reached under certain conditions (when a particular feature switch is on). For example, when the InvariantGlobalization feature switch is on (true), the GlobalizationMode/Settings class is trimmed out.

So for HybridGlobalization, is there anything we want to/can trim out if its known that HybridGlobalization is being used? From what I am understanding, HybridGlobalization is to help reduce the reliance on ICU. When its true, it doesn't make any other code completely unreachable right now does it? Or is `IcuGetLocaleInfo(..., LocalStringData ..., ...)` all supposed to be unreachable now?

Reply:
@mkhamoyan the feature switches listed here are _linker_ feature switches, i.e. where the linker replaces/hardcodes the value of the AppContext switch so that unused code can be trimmed out.

We don't need that, we have a regular AppContext switch so having it listed in this doc is confusing.

Reply:
As far as I know we just have the public docs where we could add it: https://learn.microsoft.com/en-us/dotnet/core/runtime-config/globalization